content/articles/a-sbert-based-flask-web-app-for-automatic-text-summarization/index.md

```diff
@@ -0,0 +1,204 @@
+There are a variety of ways to use SentenceTransformers in Python projects. Our preliminary work is described in this paper [Sentence-BERT:](https://arxiv.org/abs/1908.10084) Sentence Embeddings Using Siamese BERT-Networks. Using this framework, you may generate embeddings for sentences and texts in over 100 languages.
+
+We'll create a Flask web app that summarizes text automatically using BERT in this tutorial.
+### Prerequisites
+- Have a basic knowledge of python programming language.
+- Have an IDE installed, preferably [VS Code](https://code.visualstudio.com/).
+
+### Table of contents
+- [Builing the Flask web app](#builing-the-flask-web-app) 
+- [Conclusion](#conclusion)
+- [Reference](#reference)
+### Builing the Flask web app 

```

Comment:
Before we get started with the implementation, add a separate section that talks about BERT or Text summarization in-detail.